systemFlake.nix

```diff
@@ -179,6 +168,6 @@ otherArguments
   // optional checksBuilder { checks = checksBuilder pkgs; }
```

Comment:
There is an issue of builders not getting merged with actual arguments :/ I'd love to see such use-case working:
```patch
diff --git a/examples/fully-featured/flake.nix b/examples/fully-featured/flake.nix
index 5eb7ade..13071f9 100644
--- a/examples/fully-featured/flake.nix
+++ b/examples/fully-featured/flake.nix
@@ -139,6 +139,7 @@

       # All other values gets passed down to the flake
       overlay = import ./overlays;
+      packages.x86_64-linux.merge-with-packagesBuilder-test = self.pkgs.x86_64-linux.nixpkgs.hello;
       abc = 132;
       # etc
 ```
 Maybe you guys could think of some clean merge function for this? My solution would be quite lenghty

Reply:
Dumb and simple idea came to my mind:
 `mapAttrs (system: _: packages1.${system} or {} // packages2.${system} or {} ) (packages1 // packages2)` 

looks good?

Reply:
There may be the same issue with `appsBuilder` and `apps`. Should probably fix that as well.

Reply:
Yes, there are. Everything that depends on `<system>` is implemented in the same manner, but also fix is the same for all of these properties as well